### ISSUE TITLE
Checking for zero invocation is cumbersomish

### ISSUE THREAD

**Issue by ThorstenRangnau** [browncoat-ninjas/nimoy#28]
*Version*: nimoy-framework 1.0.1, python3.9

*Reproduction Test Case:*

Hi everyone,

I am really happy with nimoy :).

I found a small issue when checking number of invocations of a mock.

So I have a mock:
----
from nimoy.specification import Specification

```
with given:
    some_mock = mock.Mock()

with when:
    ...

with then:
    0 * some_mock.foo()
```
----

*Expected Behavior:*
I would expect the specification to finish with ok. However, it fails with an error.

*Actual Behavior:*
```
Traceback (most recent call last):
  File ".../lib/python3.9/site-packages/nimoy/specification.py", line 14, in helper
    return test_method(*args, **kwargs)
  File ".../specs/some_spec.py", line 60, in specify_something
    0 * some_mock.foo()
  File ".../lib/python3.9/site-packages/nimoy/specification.py", line 68, in _assert_mock
    MockAssertions().assert_mock(number_of_invocations, mock, method, *args)
  File ".../lib/python3.9/site-packages/nimoy/assertions/mocks.py", line 12, in assert_mock
    for value, expected_value in zip(mocked_method.call_args[0], args):
TypeError: 'NoneType' object is not subscriptable
```

A workaround for this is:
```
assert not some_mock.foo.called
```
It is not a big deal but for the sake of symmetry it would be nice if 0 * some_mock.foo() would work.

**Comment by noamt** [comment]
Thanks for the feedback.
The use of `0 * some_mock.foo()` totally makes sense

**Comment by noamt** [comment]
Fixed. Included in 1.0.2
